### PR TITLE
NERSC's HPSS has binaries in new locations

### PR DESCRIPTION
**Pull request by jnbellinger**

### CODE REVIEW

File: lta/nersc_mover.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 """Module to implement the NerscMover component of the Long Term Archive."""
 
 import asyncio
+from logging import Logger
```
Comment (blinkdog) [inline lta/nersc_mover.py:5]: Our integration with LGTM doesn't like that we use `from X import Y` and `import X`.
Could you remove this import and restore `Logger = logging.Logger` below?

File: lta/nersc_mover.py [hunk 2]
```diff
@@ -15,7 +16,6 @@
 from .log_format import StructuredFormatter
 from .lta_types import BundleType
 
-Logger = logging.Logger
```
Comment (blinkdog) [inline lta/nersc_mover.py:18]: This is the `Logger = logging.Logger` that I was talking about.

File: lta/nersc_retriever.py [hunk 3]
```diff
@@ -2,6 +2,7 @@
 """Module to implement the NerscRetriever component of the Long Term Archive."""
 
 import asyncio
+from logging import Logger
```
Comment (blinkdog) [inline lta/nersc_retriever.py:5]: Same thing here, need to fix it for LGTM.

File: lta/nersc_verifier.py [hunk 4]
```diff
@@ -2,6 +2,7 @@
 """Module to implement the NerscVerifier component of the Long Term Archive."""
 
 import asyncio
+from logging import Logger
```
Comment (blinkdog) [inline lta/nersc_verifier.py:5]: Same thing here, need to fix it for LGTM.

**Review by blinkdog — CHANGES_REQUESTED** [review]
The path changes for the NERSC binaries looks good.

Plumbing them through configuration may allow more flexibility for the future, but that's more complicated and higher touch, so no need to do it now. If NERSC changes up the paths again before 2022, we should consider it though.

LGTM will complain about the import change, so if we can fix those up, I think this one is ready to merge.

**Review by blinkdog — CHANGES_REQUESTED** [review]
Looks like the path for `hsi` is incorrect.

```
[pmeade@dtn01 ~]$ ls /usr/bin/hsi
ls: cannot access /usr/bin/hsi: No such file or directory
[pmeade@dtn01 ~]$ ls /usr/local/bin/hsi
/usr/local/bin/hsi
```

**Comment by jnbellinger** [comment]
icecubed@cori08:~/NEWLTA/lta> which hsi
/usr/bin/hsi

It's very odd that dtn systems should be different, but these scripts run on the cori systems

**Comment by blinkdog** [comment]
> icecubed@cori08:~/NEWLTA/lta> which hsi
> /usr/bin/hsi
> 
> It's very odd that dtn systems should be different, but these scripts run on the cori systems

Wow, that is surprising! Never mind then, carry on with `/usr/bin/hsi` :smile:

**Comment by blinkdog** [comment]
@jnbellinger If you fix the LGTM issues with the `import` and `import from`, this one is ready to merge.

**Pull request merged**